Assets/Scripts/Utility/MacroHelper.cs

```diff
@@ -767,14 +818,61 @@ private static string PlayerLastname(IMacroContextProvider mcp)
             return GetLastname(GameManager.Instance.PlayerEntity.Name);
         }
 
+        private static string DayNum(IMacroContextProvider mcp)
+        {   // %day
+            return DaggerfallUnity.Instance.WorldTime.Now.DayOfMonth.ToString();
+        }
+
+        private static string DayName(IMacroContextProvider mcp)
+        {   // %dayn
+            return DaggerfallUnity.Instance.WorldTime.Now.DayName;
+        }
+
+        private static string DayWithSuffix(IMacroContextProvider mcp)
+        {   // %days
+            return DaggerfallUnity.Instance.WorldTime.Now.DayOfMonthWithSuffix.ToString();
+        }
+
+        private static string MonthNum(IMacroContextProvider mcp)
+        {   // %mon
+            return DaggerfallUnity.Instance.WorldTime.Now.MonthOfYear.ToString();
+        }
+
+        private static string MonthName(IMacroContextProvider mcp)
+        {   // %monn
+            return DaggerfallUnity.Instance.WorldTime.Now.MonthName;
+        }
+
+        private static string YearNum(IMacroContextProvider mcp)
+        {   // %year
+            return DaggerfallUnity.Instance.WorldTime.Now.Year.ToString();
+        }
+
+        private static string TimeMin(IMacroContextProvider mcp)
+        {   // %min
+            return DaggerfallUnity.Instance.WorldTime.Now.Minute.ToString();
+        }
+
+        private static string TimeHour(IMacroContextProvider mcp)
+        {   // %hour
+            return DaggerfallUnity.Instance.WorldTime.Now.Hour.ToString();
+        }
+
+        private static string CurrentSign(IMacroContextProvider mcp)
+        {   // %sign
+            return DaggerfallUnity.Instance.WorldTime.Now.BirthSignName;
+        }
+
+        private static string CurrentSeason(IMacroContextProvider mcp)
+        {   // %sea
+            return DaggerfallUnity.Instance.WorldTime.Now.SeasonName;
+        }
+
         private static string PlayerPronoun(IMacroContextProvider mcp)
-        {   // %pg
+        {   // %pg & %pg1
```

Comment:
Did you check this was not a mistake in transferring hard strings to unity localised strings?

Reply:
It's consistent with how %g and %g1 works. As pointed out in the other comment, this is only a change in comments so maintainers aren't confused

Reply:
I've traced this down now. Seems I made a mistake when I added thse for Jay H almost 3 years ago and he never noticed. Please can you remove this change and make the change I suggest in my next review comment. Ta